test/honeysql/format_test.cljc

```diff
@@ -2,6 +2,7 @@
   (:refer-clojure :exclude [format])
   (:require [#?@(:clj [clojure.test :refer]
                  :cljs [cljs.test :refer-macros]) [deftest testing is are]]
+            honeysql.core
```

Comment:
I launched CIDER in this namespace and it wouldn't load because a bunch of stuff is referring to `honeysql.core` (e.g. lines 229-231) and it wasn't required yet.